Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,15 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+
+function toPounds(penseString) {
```

Comment:
The word is pence.

BTW It will help if you install  https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
I had installed it on my VSCode! I corrected it anyway.
Thank you.